packages/modules/manage-versions/src/components/ManageVersions/ChangesTab/ChangesTab.tsx

```diff
@@ -62,6 +61,19 @@ const ChangesTab = (props: ChangesTabProps) => {
             Header: stringsOverrides.description,
             accessor: "description",
           },
+          {
+            id: "CHANGED_FILES",
+            Header: stringsOverrides.changedFiles,
+            Cell: (props: CellProps<Changeset>) => {
+              const changedFiles =
+                props.row.original.synchronizationInfo.changedFiles;
+              return (
+                <span>
+                  {changedFiles?.length ? changedFiles.join(",") : ""}
```

Comment:
Space is needed after comma, what I mean `.join(", ")`

Reply:
change pushed